docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -2163,6 +2163,21 @@ How to **fix** it:
 - Check the write requests latency through the `Mimir / Writes` dashboard and come back to investigate the root cause of high latency (the higher the latency, the higher the number of in-flight write requests).
 - Consider scaling out the ingesters.
 
+### err-mimir-ingester-max-inflight-read-requests
+
+This error occurs when an ingester rejects a read request because the maximum in-flight requests limit has been reached.
+
+How it **works**:
+
+- The ingester has a per-instance limit on the number of in-flight read requests.
```

Comment:
Here, you hyphenate `in-flight`, whereas above you have it as one word, i.e. `inflight`. We should standardize this spelling everywhere. My preference is `in-flight`.

Reply:
Good point. There's precedent for using `inflight` in some other settings and metrics, like `max_inflight_push_requests`, so perhaps we should just go with that?